cookbooks/stac-zarr-report/index.qmd

```diff
@@ -1,5 +1,5 @@
 ---
-title: "STAC <> Zarr Report"
+title: "Overview"
```

Comment:
This doesn't seem descriptive enough.

Reply:
Good point. I was looking at the TOC when I made this change.